### ISSUE TITLE
Version of android and IOS?

### ISSUE THREAD

**Issue by BelgianCrowbar** [mendixlabs/mobile-device#5]
Is there a way that you could add the version of the android or IOS? For example Android version 8.0.0

**Comment by Andries-Smit** [comment]
The widget currently does not support it. Though its not a big feature to add as the information is available at 

https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-device/#deviceversion

Will add the feature to the backlog.

You could also give it a try yourself, add to the xml
https://github.com/mendixlabs/mobile-device/blob/master/src/MobileDevice/widget/MobileDevice.xml#L23
``` xml
        <property key="deviceVersionAttribute" type="attribute" required="false">
            <caption>Device vesrion</caption>
            <category>Data source</category>
            <description/>
            <attributeTypes>
                <attributeType name="String"/>
            </attributeTypes>
        </property>
```
Add the setter in the function:
https://github.com/mendixlabs/mobile-device/blob/master/src/MobileDevice/widget/MobileDevice.ts#L82
``` ts
        if (this.deviceVersionAttribute) {
            this.mxObject.set(this.deviceVersionAttribute, window.device ? window.device.version: "");
        }
```

**Comment by BelgianCrowbar** [comment]
Thank you for this I will try it out and let you know if it works

**Comment by BelgianCrowbar** [comment]
Works perfectly

**Comment by Andries-Smit** [comment]
Good to know. We will add the feature to the app store in the next sprint. Cheers

**Comment by jobvs** [comment]
This feature is added on master and will be available as version 1.1.0 soon!
